packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanBar.css

```diff
@@ -78,13 +104,15 @@ limitations under the License.
   background-color: rgba(0, 0, 0, 0.5);
   cursor: pointer;
   height: 60%;
-  min-width: 1px;
+  min-width: 2px;
```

Comment:
is it possible to make the bounding box of the ticks larger, instead of ticks themselves? My concern was that it was difficult to move the mouse precisely to cause a popup, going from 1px to 2px does not really help. But I did not have a problem with the visual being 1px.